type:  "babbling"
sequential:  true
data_structure:
  #depth:  "depth.yml"
  #rgb:  "rgb.yml"
  #motion:  "motion_rects.yml"
  # dataset_colorH:  "dataset_colorH.yml"
  # dataset_colorS:  "dataset_colorS.yml"
  # dataset_colorV:  "dataset_colorV.yml"
  # dataset_colorHSV:  "dataset_colorHSV.yml"
  # dataset_colorRGB:  "dataset_colorRGB.yml"
  # dataset_normalX:  "dataset_normalX.yml"
  # dataset_normalY:  "dataset_normalY.yml"
  # dataset_normalZ:  "dataset_normalZ.yml"
  # dataset_normal:  "dataset_normal.yml"
  # dataset_fpfh: "dataset_fpfh.yml"
  # dataset_colorNormalHist: "dataset_colorNormalHist.yml"
  # dataset_normalHist: "dataset_normalHist.yml"
  # dataset_normalHistLarge: "dataset_normalHistLarge.yml"
  # dataset_colorHist: "dataset_colorHist.yml"
  # dataset_colorRGBNormal: "dataset_colorRGBNormal.yml"
  # dataset_colorHSVNormal: "dataset_colorHSVNormal.yml"
  # dataset_merged: "dataset_merged.yml"
  # dataset_colorLab: "dataset_colorLab.yml"
  # dataset_colorL: "dataset_colorL.yml"
  # dataset_colora: "dataset_colora.yml"
  # dataset_colorb: "dataset_colorb.yml"
  # dataset_colorLabHist: "dataset_colorLabHist.yml"  
  dataset_meanFPFHLabHist: "dataset_meanFPFHLabHist.yml"
  # dataset_colorLabNormalHist: "dataset_colorLabNormalHist.yml"


  # gmm_archive_colorH: "gmm_archive_colorH"
  # gmm_archive_colorS: "gmm_archive_colorS"
  # gmm_archive_colorV: "gmm_archive_colorV"
  # gmm_archive_colorHSV: "gmm_archive_colorHSV"
  # gmm_archive_colorRGB: "gmm_archive_colorV"
  # gmm_archive_normalX: "gmm_archive_normalX"
  # gmm_archive_normalY: "gmm_archive_normalY"
  # gmm_archive_normalZ: "gmm_archive_normalZ"
  # gmm_archive_normal:  "gmm_archive_normal"
  # gmm_archive_colorHSV:  "gmm_archive_colorHSV"
  # gmm_archive_colorRGB:  "gmm_archive_colorRGB"
  # gmm_archive_fpfh: "gmm_archive_fpfh" 
  # gmm_archive_merged: "gmm_archive_merged"
  # gmm_archive_colorNormalHist: "gmm_archive_colorNormalHist"
  # gmm_archive_normalHist: "gmm_archive_normalHist"
  # gmm_archive_normalHistLarge: "gmm_archive_normalHistLarge"
  # gmm_archive_colorHist: "gmm_archive_colorHist"
  # gmm_archive_colorRGBNormal: "gmm_archive_colorRGBNormal"
  # gmm_archive_colorHSVNormal: "gmm_archive_colorHSVNormal"
  # gmm_archive_colorLab: "gmm_archive_colorLab"
  # gmm_archive_colorL: "gmm_archive_colorL"
  # gmm_archive_colora: "gmm_archive_colora"
  # gmm_archive_colorb: "gmm_archive_colorb"
  # gmm_archive_colorLabHist: "gmm_archive_colorLabHist"
  gmm_archive_meanFPFHLabHist: "gmm_archive_meanFPFHLabHist"
  # gmm_archive_colorLabNormalHist: "gmm_archive_colorLabNormalHist"


  #sync_dataset: "sync_dataset.yml"

  #joints_values:  "controller_feedback.yml"
  target_info: "target_info.yml"
topics:
  data:
    #sync_dataset: "/dream_babbling/sync_data"
    motion: "/dream_babbling/motion_detector_node"
    #joints_values: "/dream_babbling/controller_node/controller_node/feedback"
    # dataset_colorH: "/dream_babbling/babbling_node/dataset/colorH"
    # dataset_colorS: "/dream_babbling/babbling_node/dataset/colorS"
    # dataset_colorV: "/dream_babbling/babbling_node/dataset/colorV"
    # dataset_normalX: "/dream_babbling/babbling_node/dataset/normalX"
    # dataset_normalY: "/dream_babbling/babbling_node/dataset/normalY"
    # dataset_normalZ: "/dream_babbling/babbling_node/dataset/normalZ"
    # dataset_fpfh: "/dream_babbling/babbling_node/dataset/fpfh"
    # dataset_merged: "/dream_babbling/babbling_node/dataset/merged"
    # dataset_normal: "/dream_babbling/babbling_node/dataset/normal"
    # dataset_colorHSV: "/dream_babbling/babbling_node/dataset/colorHSV"
    # dataset_colorRGB: "/dream_babbling/babbling_node/dataset/colorRGB"
    # dataset_colorNormalHist: "/dream_babbling/babbling_node/dataset/colorNormalHist"
    # dataset_normalHist: "/dream_babbling/babbling_node/dataset/normalHist"
    # dataset_normalHistLarge: "/dream_babbling/babbling_node/dataset/normalHistLarge"
    # dataset_colorHist: "/dream_babbling/babbling_node/dataset/colorHist"
    # dataset_colorHSVNormal: "/dream_babbling/babbling_node/dataset/colorHSVNormal"
    # dataset_colorRGBNormal: "/dream_babbling/babbling_node/dataset/colorRGBNormal"
    # dataset_colorLab : "/dream_babbling/babbling_node/dataset/colorLab"
    # dataset_colorL : "/dream_babbling/babbling_node/dataset/colorL"
    # dataset_colora : "/dream_babbling/babbling_node/dataset/colora"
    # dataset_colorb : "/dream_babbling/babbling_node/dataset/colorb"
    # dataset_colorLabHist : "/dream_babbling/babbling_node/dataset/colorLabHist"
    dataset_meanFPFHLabHist : "/dream_babbling/babbling_node/dataset/meanFPFHLabHist"
    # dataset_colorLabNormalHist : "/dream_babbling/babbling_node/dataset/colorLabNormalHist"

    # gmm_archive_colorH: "/dream_babbling/babbling_node/gmm/colorH"
    # gmm_archive_colorS: "/dream_babbling/babbling_node/gmm/colorS"
    # gmm_archive_colorV: "/dream_babbling/babbling_node/gmm/colorV"
    # gmm_archive_normalX: "/dream_babbling/babbling_node/gmm/normalX"
    # gmm_archive_normalY: "/dream_babbling/babbling_node/gmm/normalY"
    # gmm_archive_normalz: "/dream_babbling/babbling_node/gmm/normalZ"
    # gmm_archive_fpfh: "/dream_babbling/babbling_node/gmm/fpfh"
    # gmm_archive_merged: "/dream_babbling/babbling_node/gmm/merged"
    # gmm_archive_normal : "/dream_babbling/babbling_node/gmm/normal"
    # gmm_archive_colorHSV : "/dream_babbling/babbling_node/gmm/colorHSV"
    # gmm_archive_colorRGB : "/dream_babbling/babbling_node/gmm/colorRGB"
    # gmm_archive_colorNormalHist: "/dream_babbling/babbling_node/gmm/colorNormalHist"
    # gmm_archive_normalHist: "/dream_babbling/babbling_node/gmm/normalHist"
    # gmm_archive_normalHistLarge: "/dream_babbling/babbling_node/gmm/normalHistLarge"
    # gmm_archive_colorHist: "/dream_babbling/babbling_node/gmm/colorHist"
    # gmm_archive_colorHSVNormal: "/dream_babbling/babbling_node/gmm/colorHSVNormal"
    # gmm_archive_colorRGBNormal: "/dream_babbling/babbling_node/gmm/colorRGBNormal"
    # gmm_archive_colorL: "/dream_babbling/babbling_node/gmm/colorL"
    # gmm_archive_colora: "/dream_babbling/babbling_node/gmm/colora"
    # gmm_archive_colorb: "/dream_babbling/babbling_node/gmm/colorb"
    # gmm_archive_colorLab: "/dream_babbling/babbling_node/gmm/colorLab"
    # gmm_archive_colorLabHist : "/dream_babbling/babbling_node/gmm/colorLabHist"
    gmm_archive_meanFPFHLabHist : "/dream_babbling/babbling_node/gmm/meanFPFHLabHist"
    # gmm_archive_colorLabNormalHist: "/dream_babbling/babbling_node/gmm/colorLabNormalHist"


    target_info: "/dream_babbling/target_info"
  servers:
    controller_server: "controller_node"
    database_server: "/dream_babbling/db_manager_node"
  services:
    motion_detector_service: "/dream_babbling/motion_detector_node/is_moving"
